rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book, :customer
+
+  def initialize(date, book, _customer)
```

Comment:
- [ ] I think you should change the text 'customer' to 'person' here. Perhaps, you made a typo here. :writing_hand: